tasks/eth/sony-002-holocene-system-config-upgrade/README.md

```diff
@@ -0,0 +1,42 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: DRAFT, NOT READY TO SIGN
```

Comment:
```suggestion
Status: READY TO SIGN
```

Reply:
The other tasks needs governance approval for the L1 absolute prestate hash, but this one will be ready to execute once merged